docs/src/_includes/components/npm_tabs.macro.html

```diff
@@ -0,0 +1,53 @@
+{%- macro npm_tabs(params) -%}
+<div class="c-tabs" data-tabs>
+   <div class="c-tabs__tablist" data-tablist hidden>
+      <button class="c-tabs__tab" data-tab>npm</button>
+      <button class="c-tabs__tab" data-tab>yarn</button>
+      <button class="c-tabs__tab" data-tab>pnpm</button>
+      <button class="c-tabs__tab" data-tab>bun</button>
+   </div>
+    <div class="c-tabs__tabpanel" data-tabpanel>
+        <h3 class="h6 c-tabs__tabpanel__title">npm</h3>
+
+        ```shell
+        {% if params.comment %}
+            # {{ params.comment | safe }}
+            {{ "\n" }}
+        {% endif -%}
+        npm {{ conversions.toNpmCommands[params.command] | safe }}
+        {% for arg in params.args %}
+            {{ conversions.toNpmArgs[arg] | safe }}
+        {% endfor %}
+        {% for package in params.packages %}
+            {{ package | safe }}
+        {% endfor %} ```
+    </div>
+   <div class="c-tabs__tabpanel" data-tabpanel>
+      <h3 class="h6 c-tabs__tabpanel__title">yarn</h3>
+
+```shell
+{% if params.comment %}# {{ params.comment | safe }}{{ "\n" }}{% endif -%}
+yarn {% if params.args.includes("--global") %}{{ conversions.toYarnArgs["--global"] | safe }} {% endif -%} {{ conversions.toYarnCommands[params.command] | safe }}{% for arg in params.args %}{% if arg !== "--global" %} {{ conversions.toYarnArgs[arg] | safe }}{% endif -%}{% endfor %}{% for package in params.packages %} {{ package.replace("@eslint/config@latest", "@eslint/config") | safe }}{% endfor %}
```

Comment:
it would be great if we could actually resolve to `params.command` when we cannot find conversions? consider if we add `yarn test` somewhere to the doc now we need to add this test resolution in `conversions.json`? it would be great if we could actually find if we have a valid conversion for it else

Reply:
I am fine with having additional data (commands) in `conversions.json`, but other than `test` and `run` command what else we can add?

Reply:
@harish-sethuraman, would like your take to move this forward!

Reply:
>it would be great if we could actually resolve to params.command when we cannot find conversions? consider if we add yarn test somewhere to the doc now we need to add this test resolution in conversions.json? it would be great if we could actually find if we have a valid conversion for it else

@harish-sethuraman, may be i didn't understand this properly, are you asking for adding more commands to `conversions.json` or updating the `npm_tabs.macro.html` to add the command directly if the command is not present in the `convesions.json`, like if `test` is not present in the data then it will be directly added to the all the commands (npm, yarn, bun, pnpm), could you please clear this to move the PR forward?

Reply:
Sorry my bad missed it. Its the next one that I was expecting `updating the npm_tabs.macro.html to add the command directly if the command is not present in the convesions.json, like if test is not present in the data then it will be directly added to the all the commands (npm, yarn, bun, pnpm`